strapi/strapi
# [Marketplace] Add plugin/provider stats

### What does it do?

Adds github stars and weekly downloads to plugin and provider cards

### How to test it?

Go to the Marketplace, you should see the statistics on the cards

This PR combines work that has already been approved on https://github.com/strapi/strapi/pull/14701 and https://github.com/strapi/strapi/pull/14737
